CREDITS

```diff
@@ -94,32 +97,41 @@ of the Coq Proof assistant during the indicated time:
 
   Bruno Barras (INRIA, 1995-now)
   Yves Bertot (INRIA, 2000-now)
-  Pierre Boutillier (INRIA-PPS, 2010-now)
+  Pierre Boutillier (INRIA-PPS, 2010-2015)
   Xavier Clerc (INRIA, 2012-2014)
+  Tej Chajed (MIT, 2016-now)
   Jacek Chrzaszcz (LRI, 1998-2003)
   Thierry Coquand (INRIA, 1985-1989)
   Pierre Corbineau (LRI, 2003-2005, Nijmegen, 2005-2008, Grenoble 1, 2008-2011)
   Cristina Cornes (INRIA, 1993-1996)
   Yann Coscoy (INRIA Sophia-Antipolis, 1995-1996)
+  Pierre Courtieu (CNAM, 2013-now)
```

Comment:
Be careful that some people changed their id at some point:
For me I changed from courtieu to Courtieu at some point

git log --pretty=format:"%an %cd" | grep "courtieu\\|Courtieu"

gives commits each years from 2006 up to now. Same for @forestjulien.

I guess 2013 is the year we switched to git. We should have detect this inconsistencies during the migration.

P.

Reply:
This information is already available in a machine readable format, although I don't know the precise invocation to use it
https://github.com/coq/coq/blob/master/.mailmap#L28

Reply:
Yeah, me neither...
Anyways, these dates issues are now fixed.